docs/BasicCommunication/ActivateApp/index.md

```diff
@@ -31,13 +31,15 @@ This request may be sent:
 ### Behavior
 
 If `level` is `FULL` or `LIMITED`: 
+
 !!! must
 
   1. Activate the application on the HMI
-    * Display [UI.Show](../../UI/Show) related parameters associated with the named `appID` in the case they were previously requested within ignition cycle
-    * Display the corresponding template in the case one was previously requested by [UI.SetDisplayLayout](../../UI/SetDisplayLayout) for that application
-    * Apply [UI.SetGlobalProperties](../../UI/SetGlobalProperties) associated with the named `appID`, if any
-    * Apply [UI.AddSubMenu](../../UI/AddSubMenu) associated with the named `appID`, if any
+  * Display [UI.Show](../../UI/Show) related parameters associated with the named `appID` in the case they were previously requested within ignition cycle
+  * Display the corresponding template in the case one was previously requested by [UI.SetDisplayLayout](../../UI/SetDisplayLayout) for that application
```

Comment:
Update this to use UI.Show instead of SetDisplayLayout. SetDisplayLayout is deprecated, Show with template configuration is used instead.

Reply:
thank you, updated in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/219/commits/255560c50476d19badaaa9e68c5d08297d6cbaac